Example/YelpAPI/YLPClient+ClientSetup.m

```diff
@@ -12,7 +12,7 @@
 @implementation YLPClient (ClientSetup)
 
 + (instancetype)newClient {
-    return [[YLPClient alloc] initWithConsumerKey:@"iVCuYETQ-P8VeICYOQS7pA" consumerSecret:@"N3GHCATpE2rQ2N9kYFw6WGDgwZA" token:@"i-T6I4uq2oh3N9JodpnP-VJ58kTXAMtw" tokenSecret:@"SYA4ajJFU-RniwW3Q4k38d9G9WY"];
+    return [[YLPClient alloc] initWithConsumerKey:@"" consumerSecret:@"" token:@"" tokenSecret:@""];
```

Comment:
You can use `#warning` here to make it clear people need to fill in their own creds, like we did in the sample: https://github.com/Yelp/yelp-api/blob/4ed773728b2babfd62ad5401721aefc5921c9420/v2/objective-c/YelpAPISample/NSURLRequest%2BOAuth.m#L17